### PR TITLE
Add schedule command cache size visualization to dashboard

### PR DESCRIPTION
**Pull request by npepinpe**
## Description

This PR adds a new graph to the processing row, showing the size of the scheduled command cache size grouped by intent.

image

I also updated the intent label values (not shown in the graph unfortunately) so they will show the intent class and name, instead of just the name. So `TRIGGER`, `TIME_OUT`, become `TimerIntent.TRIGGER` and `JobIntent.TIME_OUT` respectively.

## Related issues

related to #13870 

## Definition of Done

_Not all items need to be done depending on the issue and the pull request._

Code changes:
* [ ] The changes are backwards compatibility with previous versions
* [ ] If it fixes a bug then PRs are created to [backport](https://github.com/camunda/zeebe/compare/stable/0.24...main?expand=1&template=backport_template.md&title=[Backport%200.24]) the fix to the last two minor versions. You can trigger a backport by assigning labels (e.g. `backport stable/1.3`) to the PR, in case that fails you need to create backports manually.

Testing:
* [ ] There are unit/integration tests that verify all acceptance criterias of the issue
* [ ] New tests are written to ensure backwards compatibility with further versions
* [ ] The behavior is tested manually
* [ ] The change has been verified by a QA run
* [ ] The impact of the changes is verified by a benchmark

Documentation:
* [ ] The documentation is updated (e.g. BPMN reference, configuration, examples, get-started guides, etc.)
* [ ] If the PR changes how BPMN processes are validated (e.g. support new BPMN element) then the Camunda modeling team should be informed to adjust the BPMN linting.

Other teams:
If the change impacts another team an issue has been created for this team, explaining what they need to do to support this change.
- [ ] [Operate](https://github.com/camunda/operate/issues)
- [ ] [Tasklist](https://github.com/camunda/tasklist/issues)
- [ ] [Web Modeler](https://github.com/camunda/web-modeler/issues)
- [ ] [Desktop Modeler](https://github.com/camunda/camunda-modeler/issues)
- [ ] [Optimize](https://github.com/camunda/camunda-optimize/issues)

Please refer to our [review guidelines](https://github.com/camunda/zeebe/wiki/Pull-Requests-and-Code-Reviews#code-review-guidelines).

### CODE REVIEW

File: monitor/grafana/zeebe.json [hunk 1]
```diff
@@ -2685,6 +2697,101 @@
...
+            }
+          },
+          "targets": [
+            {
+              "datasource": {
+                "type": "prometheus",
+                "uid": "${DS_PROMETHEUS}"
+              },
+              "editorMode": "code",
+              "expr": "sum (zeebe_stream_processor_scheduled_command_cache_size{cluster=~\"$cluster\", namespace=~\"$namespace\", pod=~\"$pod\", partitionId=~\"$partition\"}) by (intent)",
```
Comment (Zelldon) [inline monitor/grafana/zeebe.json:2786]: Do we want to sum here 🤔

Comment (npepinpe) [reply monitor/grafana/zeebe.json:2786]: Why not? We can filter by pod/partition via the dashboard variables.

Comment (Zelldon) [reply monitor/grafana/zeebe.json:2786]: I guess we have/had two different expectations of what to see. I have expected exactly that: to see all pods and partitions the cache size yes. Similar to what I had in the benchmark I did. I don't know feels to me more natural to see for all what the their current cache sizes are (I guess this is my personal FOMO kink, how I would see that the follower still has a cache of 20 😅) 

Anyway maybe you want to change it to max, so we rule out other influence factors 🤷🏼

Comment (npepinpe) [reply monitor/grafana/zeebe.json:2786]: I don't have strong preference for summing, it was more natural for me, but it's not a huge deal to have it split.

Comment (npepinpe) [reply monitor/grafana/zeebe.json]: Switched to max by pod, partition, and intent.

**Review by ChrisKujawa — APPROVED** [review]

**Comment by npepinpe** [comment]
bors merge

**Pull request merged**